coda-automation/services/faucet/tiny.py

```diff
@@ -0,0 +1,227 @@
+import discord
```

Comment:
I'm pretty sure these services have been deprecated for a single bots impl using Reason so should be ok to clean up here also.

src: https://github.com/MinaProtocol/mina/tree/develop/frontend/bot
legacy Docker image: https://hub.docker.com/r/codaprotocol/coda-bots
current Docker image: https://hub.docker.com/r/codaprotocol/bot